modules/ref-dashboard-configuration-options.adoc

```diff
@@ -11,41 +11,61 @@ The {productname-short} dashboard includes a set of core features enabled by def
+*Note:* Do not edit the default value; this feature is not available in this version of {productname-short}.
+// When the feature is supported, uncomment the following two lines, and delete the above line and this line. 
+//Hides the *Settings → Connection types* option in the dashboard navigation menu. 
+//To show this menu option, set the value to `false`.
 | `dashboardConfig:
-disableBYONImageStream` | `false` | Shows the *Settings → Notebook images* option in the dashboard navigation menu. To hide this menu option, set the value to `false`.
+disableCustomServingRuntimes` | `false` | Shows the *Settings → Serving runtimes* option in the dashboard navigation menu. To hide this menu option, set the value to `true`. 
 | `dashboardConfig:
-disableISVBadges` | `false` | Shows the label on a tile that indicates whether the application is “{org-name} managed”, “Partner managed”, or “Self-managed”. To hide these labels, set the value to `true`. 
+disableDistributedWorkloads` | `false` | Shows the *Distributed Workload Metrics* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
-disableUserManagement` | `false` | Shows the *Settings → User management* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
+disableHome` | `false` | Shows the *Home* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
-disableProjects` | `false` | Shows the *Data Science Projects* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
- | `dashboardConfig:
- disablePipelines` | `false` | Shows the *Data Science Pipelines* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
+disableInfo` | `false` | On the *Applications → Explore* page, when a user clicks on an application tile, an information panel opens with more details about the application. To disable the information panel for all applications on the *Applications → Explore* page , set the value to `true`.
 | `dashboardConfig:
-disableModelServing` | `false` | Shows the *Model Serving* option in the dashboard navigation menu and in the list of components for the data science projects. To hide *Model Serving* from the dashboard navigation menu and from the list of components for data science projects, set the value to `true`. 
+disableISVBadges` | `false` | Shows the label on a tile that indicates whether the application is “{org-name} managed”, “Partner managed”, or “Self-managed”. To hide these labels, set the value to `true`. 
 | `dashboardConfig:
-disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users. To prevent users from sharing data science projects, set the value to `true`.
+disableKServe` | `false` | Enables the ability to select KServe as a Serving Platform. To disable this ability, set the value to `true`.
```

Comment:
This should just be `serving platform` or `model-serving platform`, doesn't need to be capitalized. The same is true for lines 40, 44, and 50.

Reply:
Thank you @syaseen-rh! Updated on lines 38, 44, and 50. Didn't see any occurrence on line 40? 

However, I did want to double-check that the current text on line 40 is correct, or should it be rephrased? 
In particular, can you please confirm that "auth" is the correct term? (maybe that's why you mentioned line 40?)

Reply:
@bredamc apologies, I just noticed that you removed the line 40 occurrence, so all good!
`disableKServe` | `false` | Enables the ability to select KServe as a Serving Platform. To disable this ability, set the value to `true`.`